src/App.js

```diff
@@ -17,19 +17,19 @@ let todoList = [
 
   function App() {
     return (
-      <>
+      <React.Fragment>
       <h2>Todo List</h2>
```

Comment:
I hope this doesn't feel too nitpicky! The third step in [these instructions](https://github.com/Code-the-Dream-School/react/wiki/Lesson-1.1#react-components-and-jsx) want the heading to level to be 1, and it's 2 here.